layouts/partials/common/opengraph.html

```diff
@@ -47,14 +47,14 @@
 {{ end }}{{ end }}
 {{- end }}
 
-{{- /* Deprecate site.Social.facebook_admin in favor of site.Params.social.facebook_admin */}}
+{{- /* Deprecate site.Social.facebook_admin in favor of */}}
```

Comment:
@Noam-Alum should this comment be removed entirely, or should it be updated to reflect the new one? so

`{{- /* Deprecate site.Social.facebook_admin in favor of site.Params.social.facebook_admin */}}`

Reply:
I think removing the comment would be the best choice as its not important to note a deprecated parameter. (Also anyone can see the file changes in GitHub if for reasons beyond me this would cause any harm in the future)